src/librustc_driver/lib.rs

```diff
@@ -519,32 +515,35 @@ fn usage(verbose: bool, include_unstable_options: bool) {
     } else {
         config::rustc_short_optgroups()
     };
-    let groups : Vec<_> = groups.into_iter()
-        .filter(|x| include_unstable_options || x.is_stable())
-        .map(|x|x.opt_group)
-        .collect();
+    let groups: Vec<_> = groups.into_iter()
+                               .filter(|x| include_unstable_options || x.is_stable())
+                               .map(|x| x.opt_group)
+                               .collect();
     let message = format!("Usage: rustc [OPTIONS] INPUT");
     let extra_help = if verbose {
         ""
     } else {
         "\n    --help -v           Print the full set of options rustc accepts"
     };
-    println!("{}\n\
-Additional help:
+    println!("{}\nAdditional help:
     -C help             Print codegen options
-    -W help             Print 'lint' options and default settings
-    -Z help             Print internal options for debugging rustc{}\n",
-              getopts::usage(&message, &groups),
-              extra_help);
+    -W help             \
+              Print 'lint' options and default settings
+    -Z help             Print internal \
+              options for debugging rustc{}\n",
+             getopts::usage(&message, &groups),
+             extra_help);
```

Comment:
Function probably needs a `#[rustfmt_skip]` so that it leaves the string lit alone.